Broken build on Ubuntu 20.04 (565d1e27d75a51223b8f48b618ec913320f836e0)

Tested commit: 565d1e27d75a51223b8f48b618ec913320f836e0
This seems unrelated to the previous problem I was having

```
In file included from ../Source/Tests/LinuxSyscalls/x32/IoctlEmulation.cpp:1:
In file included from ../Source/Tests/LinuxSyscalls/x32/Ioctl/drm.h:5:
In file included from ../Source/Tests/LinuxSyscalls/x32/Types.h:12:
In file included from /usr/include/x86_64-linux-gnu/asm/ipcbuf.h:1:
/include/asm-generic/ipcbuf.h:21:2: error: unknown type name '__kernel_key_t'
        __kernel_key_t          key;
        ^
/include/asm-generic/ipcbuf.h:22:2: error: unknown type name '__kernel_uid32_t'
        __kernel_uid32_t        uid;
        ^
/include/asm-generic/ipcbuf.h:23:2: error: unknown type name '__kernel_gid32_t'
        __kernel_gid32_t        gid;
        ^
/include/asm-generic/ipcbuf.h:24:2: error: unknown type name '__kernel_uid32_t'
        __kernel_uid32_t        cuid;
        ^
/include/asm-generic/ipcbuf.h:25:2: error: unknown type name '__kernel_gid32_t'
        __kernel_gid32_t        cgid;
        ^
/include/asm-generic/ipcbuf.h:26:2: error: unknown type name '__kernel_mode_t'
        __kernel_mode_t         mode;
        ^
/include/asm-generic/ipcbuf.h:28:35: error: use of undeclared identifier '__kernel_mode_t'
        unsigned char           __pad1[4 - sizeof(__kernel_mode_t)];
                                                  ^
/include/asm-generic/ipcbuf.h:31:2: error: unknown type name '__kernel_ulong_t'
        __kernel_ulong_t        __unused1;
        ^
/include/asm-generic/ipcbuf.h:32:2: error: unknown type name '__kernel_ulong_t'
        __kernel_ulong_t        __unused2;
        ^
In file included from ../Source/Tests/LinuxSyscalls/x32/IoctlEmulation.cpp:1:
In file included from ../Source/Tests/LinuxSyscalls/x32/Ioctl/drm.h:5:
In file included from ../Source/Tests/LinuxSyscalls/x32/Types.h:13:
In file included from /usr/include/x86_64-linux-gnu/asm/shmbuf.h:6:
/include/asm-generic/shmbuf.h:29:2: error: unknown type name '__kernel_time_t'
        __kernel_time_t         shm_atime;      /* last attach time */
        ^
/include/asm-generic/shmbuf.h:30:2: error: unknown type name '__kernel_time_t'
        __kernel_time_t         shm_dtime;      /* last detach time */
        ^
/include/asm-generic/shmbuf.h:31:2: error: unknown type name '__kernel_time_t'
        __kernel_time_t         shm_ctime;      /* last change time */
        ^
/include/asm-generic/shmbuf.h:40:2: error: unknown type name '__kernel_pid_t'
        __kernel_pid_t          shm_cpid;       /* pid of creator */
        ^
/include/asm-generic/shmbuf.h:41:2: error: unknown type name '__kernel_pid_t'
        __kernel_pid_t          shm_lpid;       /* pid of last operator */

```

--- a/Source/Tests/LinuxSyscalls/x32/Types.h
+++ b/Source/Tests/LinuxSyscalls/x32/Types.h
@@ -16,6 +16,7 @@ $end_info$
 #include <cstring>
 #include <fcntl.h>
 #include <limits>
+#include <linux/types.h>
 #include <sys/ipc.h>
 #include <mqueue.h>
 #include <signal.h>

